source/reference/verification.txt

```diff
@@ -30,12 +35,10 @@ destination cluster.
 
 Verification methods:
 
+- :ref:`c2c-verify-with-embedded-verifier`
```

Comment:
I think we want to make it clear that this one is preferred if the limitations can be worked with

Reply:
Fixed